Source/React.cpp

```diff
@@ -1,5 +1,35 @@
 #include "React.H"
 
+void
+PeleC::set_typical_values_chem()
+{
+
```

Comment:
Since this only gets called on init/restart. Why not just move the pp query's here and then move the new PeleC class members into here and just make them normal variables? That way everything is local to one spot.

Reply:
It gets called after regrid and every reset_typ_vals_int interval.

Reply:
ah ok nvm then